backend/tests/test_auth_api.py

```
@@ -2,160 +1,70 @@
-        "message": "Verification email sent. Please check your inbox."
-    }
-
-    user = db_session.query(User).filter(User.email == user_data["email"]).first()
-    assert user is not None
-    assert user.email == user_data["email"]
+    data = response.json()
+    assert "access_token" in data
+    assert data["token_type"] == "bearer"
 
 
+def test_login_user(override_get_db, test_db):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_login_user`

Add return type annotation: `None`

(ANN201)